site/docs/pages/identity/avatar.mdx

```diff
@@ -1,10 +1,11 @@
 import { Avatar, Badge, Identity } from '@coinbase/onchainkit/identity';
 import App from '../../components/App';
+import { base } from 'viem/chains';
 
 # `<Avatar />`
 
-The `Avatar` component is used to display ENS avatar associated with Ethereum addresses.
-When an ENS avatar is not available, it defaults to blue color avatar.
+The `Avatar` component is used to display ENS or Basenames avatar associated with Ethereum addresses.
```

Comment:
Can we link `Basenames` anywhere? maybe the official Basename site, so we add more context what those are.

Reply:
`Basenames` will now link to https://www.base.org/names

<img width="665" alt="Screenshot 2024-08-12 at 3 04 53 PM" src="https://github.com/user-attachments/assets/9ea90015-055e-4445-b127-e95120bf7d99">